backend/controllers/jobPostingsController.js

```diff
@@ -27,7 +27,7 @@ const getJobPostingById = async (req, res) => {
 
 // create a new job posting
 const createJobPosting = async (req, res) => {
-    const { company, jobTitle, location, jobType, details, noDegreeMentioned, benefits } = req.body;
```

Comment:
dont think the imported mongo variable is used anywhere in this file. If so please remove